hasp/wrapper.py

```diff
@@ -1219,10 +1222,13 @@ def call_main():
     parser.add_argument("-g", "--grating_table",
                         default=None,
                         help="Name of grating priority file [None]")
+    parser.add_argument("-n", "--target_name",
+                        default=None,
+                        help="Target name [use directory name]")
```

Comment:
This is for cross-program products only, so maybe
```suggestion
                        help="Target name (cross-program products only) [use directory name]")
```